### ISSUE TITLE
Jobs failing to submit for longer than 3 hours

### ISSUE THREAD

**Issue by moldach** [Snakemake-Profiles/slurm#56]
I'm running into an issue on my HPC where jobs are not submitted to the scheduler for longer than **3 hours and 20 minutes**. 

I'm trying to figure out if this is any of the following:

* An error caused by `Snakemake-Profiles/slurm`
* An error caused by `Snakemake`
* An potential bug in the routing engine of the HPC

## Installation/Setup

```
$ module load python/3.8.2
$ python3 -m venv ~/bin/python3.8.2_venv
$ source ~/bin/python3.8.2_venv/bin/activate
(python3.8.2_venv) $ pip install cookiecutter
(python3.8.2_venv) $ mkdir -p ~/.config/snakemake
(python3.8.2_venv) $ cd ~/.config/snakemake
(python3.8.2_venv) $ cookiecutter https://github.com/Snakemake-Profile                                                                                            s/slurm.git
You've downloaded /home/moldach/.cookiecutters/slurm before. Is it okay to delete and re-download it                                                                                            ? [yes]: yes
profile_name [slurm]:
sbatch_defaults []:
cluster_config []:
Select advanced_argument_conversion:
1 - no
2 - yes
Choose from 1, 2 [1]: 1
(python3.8.2_venv) $ pip install snakemake
```

## `Snakemake` version
```
(python3.8.2_venv) $ snakemake --version
5.26.1
```

## `Snakefile`

```
rule all:
    input:
        "test_60min",
        "test_200min"

rule test_short_time:
    params:
        test1="test1"
    output:
        "test_60min"
    resources:
        mem=1000,
        time=60
    shell: """
        sleep 3600
        touch {output}
        """

rule test_long_time:
    params:
        test2="test2"
    output:
        "test_200min"
    resources:
        mem=1000,
        time=200
    shell: """
        sleep 12000
        touch  {output}
        """
```

## Submit job

```
(python3.8.2_venv) $ bash -c "nohup snakemake --profile slurm2 &"
```

## Check on how long it's submitted for

Quickly  I ran `squeue` to see how long it was given:

```
(python3.8.2_venv) $ squeue -u moldach
          JOBID     USER              ACCOUNT           NAME  ST  TIME_LEFT NODES CPUS TRES_
       39447461  moldach     def-mtarailo_cpu snakejob.test_   R      59:43     1    1
       39447462  moldach     def-mtarailo_cpu snakejob.test_   R    3:19:43     1    1
```

more detailed information:

```
(python3.8.2_venv) [moldach@gra-login3 SLURM-TEST]$ scontrol show jobid -dd 39447462
JobId=39447462 JobName=snakejob.test_long_time.2.sh
   UserId=moldach(3019658) GroupId=moldach(3019658) MCS_label=N/A
   Priority=3064061 Nice=0 Account=def-mtarailo_cpu QOS=normal
   JobState=RUNNING Reason=None Dependency=(null)
   Requeue=0 Restarts=0 BatchFlag=1 Reboot=0 ExitCode=0:0
   DerivedExitCode=0:0
   RunTime=00:23:12 TimeLimit=03:20:00 TimeMin=N/A
   SubmitTime=2020-10-14T20:43:56 EligibleTime=2020-10-14T20:43:56
   AccrueTime=2020-10-14T20:43:56
   StartTime=2020-10-14T20:43:57 EndTime=2020-10-15T00:03:57 Deadline=N/A
   SuspendTime=None SecsPreSuspend=0 LastSchedEval=2020-10-14T20:43:57
   Partition=cpubase_bycore_b2 AllocNode:Sid=gra-login3:17953
   ReqNodeList=(null) ExcNodeList=(null)
   NodeList=gra704
   BatchHost=gra704
   NumNodes=1 NumCPUs=1 NumTasks=1 CPUs/Task=1 ReqB:S:C:T=0:0:*:*
   TRES=cpu=1,mem=1000M,node=1,billing=1
   Socks/Node=* NtasksPerN:B:S:C=0:0:*:* CoreSpec=*
     Nodes=gra704 CPU_IDs=19 Mem=1000 GRES=
   MinCPUsNode=1 MinMemoryNode=1000M MinTmpDiskNode=0
   Features=[broadwell|skylake|cascade] DelayBoot=00:00:00
   OverSubscribe=OK Contiguous=0 Licenses=(null) Network=(null)
   Command=/scratch/moldach/SLURM-TEST/.snakemake/tmp.45pg9218/snakejob.test_long_time.2.sh
   WorkDir=/scratch/moldach/SLURM-TEST
   StdErr=/scratch/moldach/SLURM-TEST/slurm-39447462.out
   StdIn=/dev/null
   StdOut=/scratch/moldach/SLURM-TEST/slurm-39447462.out
   Power=
```

**Comment by johnstonmj** [comment]
Hi Matthew,

I'm not sure I see the failure in the above details.
Jobs submitted with `time=60` and `time=200` both appear to be running with `squeue`.
Are you saying that jobs submitted with `time=400` (or any number >200) will fail? If so, can you show the details?

Cheers

**Comment by moldach** [comment]
Hi @johnstonmj I apologize, the above details were confusing/incomplete. In fact there was no error above... `time=200` is `3:20` so you're correct.

I was trying to run a 7 hour script on another HPC and it would only submit for `3 hours` - I got hasty an didn't notice that on this other HPC it submitted properly.

Perhaps there was something I did wrong when I configured my `profile`?
In any case, deleting  `~/.config/snakemake` and re-installing seemed to solve the issue.
